### PR TITLE
Advanced Mapping: namespace platform event channel name

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes
- Fix platform event listener component's channel name namespacing

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by psadie-sfdo** [comment]
**lurch: attach W-035056

**Comment by psadie-sfdo** [comment]
**lurch: attach W-035056

**Comment by psadie-sfdo** [comment]
**lurch: attach W-035056

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001C8x2QAC" target="_blank">W-035056</a>

File: src/classes/UTIL_Namespace.cls [hunk 1]
```diff
@@ -45,6 +45,7 @@ public class UTIL_Namespace {
     * @description Finds the namespace for the current context.
     * @return string The current namespace as a string, or a blank string if we're not in a namespaced context.
     ********************************************************************************************************/
+    @AuraEnabled
```
Comment (ptewson-sfdo) [inline src/classes/UTIL_Namespace.cls]: @force2b - Do we have a policy of avoiding AuraEnabled in our util classes? Should we support this with a pass-through method in the LWC component controller?

Comment (force2b) [reply src/classes/UTIL_Namespace.cls]: I'm not sure there's an official policy, but I wouldn't recommend it. Your suggestion of a pass through would be better.

**Pull request merged**